autotx/utils/ethereum/networks.py

```diff
@@ -0,0 +1,128 @@
+from dataclasses import dataclass
+import json
+from typing import Union
+import requests
+from gnosis.eth import EthereumNetwork
+from web3 import Web3
+
+from autotx.utils.ethereum.constants import NATIVE_TOKEN_ADDRESS
+
+KLEROS_TOKENS_LIST = "https://t2crtokens.eth.link/"
+COINGECKO_TOKENS_LIST = "https://tokens.coingecko.com/uniswap/all.json"
```

Comment:
Why not just fetch these at build-time and load them from a JSON file? Could also embed them in a python source file.

Reply:
thats a good point... will do that

Reply:
implemented here https://github.com/polywrap/AutoTx/pull/127 lmk your thoughts ser @dOrgJelli